shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.h

```diff
@@ -16,6 +16,32 @@
 #include "flutter/shell/platform/darwin/ios/ios_context.h"
 #include "third_party/skia/include/core/SkPictureRecorder.h"
 
+// A UIView acts as a clipping mask for the |ChildClippingView|.
+//
+// On |DrawRect:|, this view performs a series of clipping operations and sets the alpha channel
```

Comment:
should this interface contain the virtual `drawRect` method?

Reply:
This View is an OBJC class, drawRect is inherited from the UIView. So I don't think any virtual keyword is necessary here.

Reply:
sg. do you where I can read about the doc format? I'm not sure what `|DrawRect:|` means, but I can see it means what you described.

Reply:
https://developer.apple.com/documentation/uikit/uiview/1622529-drawrect

Reply:
I mean the notation. It reads like a class to me, but I’m not sure what the colon symbol is used for.

Reply:
ah you are right, let me fix that